pages/common/pulumi-gen-completion.md

```diff
@@ -0,0 +1,9 @@
+# pulumi gen-completion
+
+> Generate completion scripts for the Pulumi CLI.
+> Supported shells are bash, zsh, fish.
+> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_gen-completion/>.
+
+- Generate completion scripts:
+
+`pulumi gen-completion {{SHELL}}`
```

Comment:
```suggestion
`pulumi gen-completion {{bash|zsh|fish}}`
```

Reply:
Haha, literally changed it last second from what you've suggested to what I've commited. =)